jasmine/src/bowling.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+var BowlingScorecard = function() {
+  this.knockedDownPins = 0;
+  this.frame = []
+  this.total = []
+  this.frameCount = 0
+};
+
+BowlingScorecard.prototype.getCurrentRollScore = function() {
```

Comment:
I see that you use prototypes when defining functions, and this within the constructor when defining properties. Why did you decide on this distinction?